pkg/trace/traceutil/azure.go

```diff
@@ -50,16 +52,17 @@ func getAppServicesTags(getenv func(string) string) map[string]string {
 	resourceGroup := getenv("WEBSITE_RESOURCE_GROUP")
 	instanceID := getEnvOrUnknown("WEBSITE_INSTANCE_ID", getenv)
 	computerName := getEnvOrUnknown("COMPUTERNAME", getenv)
-	currentRuntime := getRuntime(getenv)
+	extensionVersion := getenv("DD_AAS_EXTENSION_VERSION")
```

Comment:
🗒️  not intended for this PR as I don't want to bog you down, but I wanted to point out I'm noticing a ton of constant values being assigned here and there for our AAS implementation. We might start to lose track of what's what so we should create a `constants.go` file in a separate PR and move all of our static values into it